hack/update-codegen.sh

```diff
@@ -24,24 +24,28 @@ CODEGEN_PKG=$($GO_CMD list -m -f "{{.Dir}}" k8s.io/code-generator)
 
 cd $(dirname ${BASH_SOURCE[0]})/..
 
-chmod +x "${CODEGEN_PKG}/generate-internal-groups.sh"
-chmod +x "${CODEGEN_PKG}/generate-groups.sh"
-"${CODEGEN_PKG}/generate-groups.sh" \
-  applyconfiguration,client,lister,informer \
-  sigs.k8s.io/kueue/client-go \
-  sigs.k8s.io/kueue/apis \
-  kueue:v1beta1 \
-  --go-header-file ${KUEUE_ROOT}/hack/boilerplate.go.txt
+chmod +x "${CODEGEN_PKG}/kube_codegen.sh"
+source "${CODEGEN_PKG}/kube_codegen.sh"
+
+kube::codegen::gen_helpers \
+  --input-pkg-root sigs.k8s.io/kueue/apis \
+  --output-base "$(dirname "${BASH_SOURCE[0]}")/../../.." \
```

Comment:
Maybe we could make it relative to `${KUEUE_ROOT}`? Since we anyway use it for boilerplate param.

Reply:
+1 

use `"${KUEUE_ROOT}/client-go"`.

Reply:
Using ```"${KUEUE_ROOT}/client-go"``` as ```--output-base``` changes the name of the generated package, so imho it's not the best idea. However I'll make it the ```--output-base``` relative to ```"${KUEUE_ROOT}/client-go"```